src/App.js

```diff
@@ -13,13 +13,15 @@ import DonorDetails from "./components/DonorDetails";
 import SearchResults from "./components/SearchResults";
  
 function App() {
-  
   return (
     <div className="App">
       <Router history={history}>
         <Switch>
           <Route path="/login">
-            <Login />
+            <Login/>
+          </Route>
+          <Route path="/reset-password">
+            <Login resetPasswordStatus= {true} />
```

Comment:
Since this prop is boolean, you can leave out the `= {true}`

In face, you are calling it a different name in Login.js, why not just follow that? I think it's a more accurate name.
`<Login resetPasswordMode />`